proto/collectd.proto

```diff
@@ -25,35 +25,32 @@
 syntax = "proto3";
 
 package collectd;
+option go_package = "collectd.org/rpc/proto";
 
 import "types.proto";
 
 service Collectd {
-	// Dispatch collected values to collectd.
-	rpc DispatchValues(DispatchValuesRequest) returns (DispatchValuesReply);
+  // Query a list of values available from collectd's value cache.
+  rpc QueryValues(QueryValuesRequest) returns (stream QueryValuesResponse);
 
-	// Query a list of values available from collectd's value cache.
-	rpc QueryValues(QueryValuesRequest) returns (QueryValuesReply);
+  // DispatchValues sends a stream of ValueLists to the server.
```

Comment:
I've reworded this to be phrased from the POV of the server to match QueryValues' description.